NOnion/Crypto/HiddenServicesCipher.fs

```diff
@@ -170,11 +170,10 @@ module HiddenServicesCipher =
         let macKey = hsKeys |> Array.skip 32 |> Array.take 32
 
         let cipher = TorStreamCipher(encKey, None)
-        let digest = data |> CalculateMacWithSHA3256 macKey
 
         let encryptedInnerData = data |> cipher.Encrypt
 
-        (encryptedInnerData, digest)
+        encryptedInnerData, macKey
```

Comment:
nit abt commit message: "fix introduce"? is it fix, or is it introduce?

Reply:
Fixed

Reply:
Fixed